Rakefile

```
@@ -343,3 +343,15 @@ begin
 rescue LoadError
   # Optional group in bundler
 end
+
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new(:rubocop) do |task|
+  # These make the rubocop experience maybe slightly less terrible
+  task.options = ['-D', '-S', '-E']
+
+  # Use Rubocop's Github Actions formatter if possible
+  if ENV['GITHUB_ACTIONS'] == 'true'
+    rubocop_spec = Gem::Specification.find_by_name('rubocop')
+    task.formatters << 'github' if Gem::Version.new(rubocop_spec.version) >= Gem::Version.new('1.2')
```

Since you require a new enough version, there's no need to try to be compatible with old versions.
```suggestion
    task.formatters << 'github'
```

---

done